src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1164,6 +1164,24 @@ public void updateFeedbackSession(FeedbackSessionAttributes newSession)
 
         fsDb.updateFeedbackSession(newSession);
     }
+
+    public void updateSessionResponseRateForDeletedStudentResponse(String email, String sessionName,
+            String courseId) throws InvalidParametersException, EntityDoesNotExistException {
+        if (noRemainingResponsesFromStudent(email, sessionName, courseId)) {
+            deleteStudentFromRespondentList(email, sessionName, courseId);
+        }
+    }
+
+    private boolean noRemainingResponsesFromStudent(String studentEmail, String sessionName, String courseId) {
```

Comment:
We should avoid negation of things in method names. This method should be reimplemented as `hasRemainingResponsesFromStudent`, I think? And the return value negated in where it is used